### ISSUE TITLE
I need mvc back

### ISSUE THREAD

**Issue by ultrasound1372** [kuma4649/mml2vgm#151]
The command line compiler (mvc.exe) seems to no longer be included in the latest releases. As a blind person I cannot use the IDE due to the accessibility of the text control, so I normally just write my mml in notepad and compile it with mvc. I attempted to use mml2vgm.exe instead, but it seems to be not accessible to my screen reader. There are no menus like file, edit etc, that might allow me to open a file and compile it. I was able to use advance navigation to find a tool bar, but the buttons on it are just called 'tool bar button 1' and the like, they don't have proper labels for me to know what they do. Could you see to including mvc.exe in future releases so that I may continue to use mml2vgm?
***
コマンドラインコンパイラ（mvc.exe）は最新のリリースにはもう含まれていないようです。全盲の私は、テキストコントロールのアクセシビリティのためにIDEを使うことができないので、通常はメモ帳にmmlを書いてmvcでコンパイルしています。代わりにmml2vgm.exeを使おうとしましたが、私のスクリーンリーダーではアクセスできないようです。ファイルを開いてコンパイルできるような、ファイルや編集などのメニューがないのです。アドバンス・ナビゲーションを使ってツールバーを見つけることはできましたが、その上のボタンは「ツールバーボタン1」などと呼ばれているだけで、何をするのかがわかるような適切なラベルがついていません。私がmml2vgmを使い続けることができるように、将来のリリースにmvc.exeを含めるように見てもらえますか？
